### PR TITLE
test: solve test cov

### PR DESCRIPTION
**Pull request by li-jia-nan**
[中文版模板 / Chinese template](https://github.com/ant-design/ant-design/blob/master/.github/PULL_REQUEST_TEMPLATE/pr_cn.md?plain=1)

### 🤔 This is a ...

- [ ] New feature
- [ ] Bug fix
- [ ] Site / documentation update
- [ ] Demo update
- [ ] Component style update
- [ ] TypeScript definition update
- [ ] Bundle size optimization
- [ ] Performance optimization
- [ ] Enhancement feature
- [ ] Internationalization
- [ ] Refactoring
- [ ] Code style optimization
- [x] Test Case
- [ ] Branch merge
- [ ] Workflow
- [ ] Other (about what?)

### 📝 Changelog

| Language   | Changelog |
| ---------- | --------- |
| 🇺🇸 English | test: solve test cov |
| 🇨🇳 Chinese | test: solve test cov |

### ☑️ Self-Check before Merge

⚠️ Please check all items below before requesting a reviewing. ⚠️

- [x] Doc is updated/provided or not needed
- [x] Demo is updated/provided or not needed
- [x] TypeScript definition is updated/provided or not needed
- [x] Changelog is provided or not needed

### CODE REVIEW

File: components/typography/Base/CopyBtn.tsx [hunk 1]
```diff
@@ -20,12 +20,22 @@ export interface CopyBtnProps extends Omit<CopyConfig, 'onCopy'> {
...
 const CopyBtn: React.FC<CopyBtnProps> = (props) => {
-  const { prefixCls, copied, locale = {}, onCopy, iconOnly, tooltips, icon, loading } = props;
+  const {
+    prefixCls,
+    copied,
+    locale,
+    iconOnly,
+    tooltips,
+    icon,
+    loading: btnLoading,
```
Comment (li-jia-nan) [inline components/typography/Base/CopyBtn.tsx:30]: 这里重命名是为了强制换行，验证一下覆盖率是否会挂

Comment (li-jia-nan) [reply components/typography/Base/CopyBtn.tsx:30]: CI 跑完了，看起来 cov 没问题

Comment (nova1751) [reply components/typography/Base/CopyBtn.tsx:30]: 考虑把`tabIndex`之类的都给解构一下吗，感觉这样代码结构更统一一些

Comment (li-jia-nan) [reply components/typography/Base/CopyBtn.tsx:30]: 不是解构了吗

Comment (nova1751) [reply components/typography/Base/CopyBtn.tsx:30]: 噢噢，我的，看漏了

Comment (li-jia-nan) [reply components/typography/Base/CopyBtn.tsx:30]: = {} 去掉了，不够长

**Review by afc163 — APPROVED** [review]

**Pull request merged**